package.json

```
@@ -92,6 +92,18 @@
                     "scope": "resource",
                     "type": "array"
                 },
+                "black-formatter.showNotification": {
+                    "default": "on-warn",
+                    "description": "When extension is allowed to notifications messages in UI.",
+                    "enum": [
+                        "on-crash",
+                        "on-error",
+                        "on-warn",
+                        "on-message"
+                    ],
+                    "scope": "resource",
```

```suggestion
                    "scope": "machine",
```
I don't think this should be `resource`, this setting need not have workspace folder level of granularity. It can be `machine` where it can be handled in both local and remote scenarios.